apps/server/src/infra/sync/strategy/tsp/tsp-oauth-data.mapper.ts

```diff
@@ -1,18 +1,18 @@
+import { Injectable } from '@nestjs/common';
+import { BadDataLoggableException } from '@modules/provisioning/loggable';
+import { System } from '@modules/system';
+import { RoleName } from '@shared/domain/interface';
+import { SystemProvisioningStrategy } from '@shared/domain/interface/system-provisioning.strategy';
+import { School } from '@modules/school';
 import {
 	ExternalClassDto,
 	ExternalSchoolDto,
 	ExternalUserDto,
 	OauthDataDto,
 	ProvisioningSystemDto,
 } from '@modules/provisioning';
-import { School } from '@modules/school';
-import { System } from '@modules/system';
-import { Injectable } from '@nestjs/common';
-import { RoleName } from '@shared/domain/interface';
-import { SystemProvisioningStrategy } from '@shared/domain/interface/system-provisioning.strategy';
 import { Logger } from '@src/core/logger';
-import { RobjExportKlasse, RobjExportLehrer, RobjExportSchueler } from '@src/infra/tsp-client';
-import { BadDataLoggableException } from '@src/modules/provisioning/loggable';
+import { RobjExportKlasse, RobjExportLehrer, RobjExportSchueler } from '../../../tsp-client';
```

Comment:
Not the same module.
```suggestion
import { RobjExportKlasse, RobjExportLehrer, RobjExportSchueler } from '@src/infra/tsp-client';
```